doc/source/cluster/kubernetes/examples/rayjob-batch-inference-example.md

```diff
@@ -0,0 +1,159 @@
+(kuberay-batch-inference-example)=
+
+# RayJob Batch Inference Example
+
+This page demonstrates how to use the RayJob custom resource to run a batch inference job on a Ray cluster.
+
+We will use an image classification workload. The example is based on <https://docs.ray.io/en/latest/data/examples/huggingface_vit_batch_prediction.html>. Please see that page for a full explanation of the code.
+
+## Prerequisites
+
+You must have a Kubernetes cluster running and `kubectl` configured to use it, and GPUs available. We provide a brief tutorial for setting up the necessary GPUs on Google Kubernetes Engine (GKE), but you can use any Kubernetes cluster with GPUs.
+
+## Step 0: Create a Kubernetes cluster on GKE (Optional)
+
+If you already have a Kubernetes cluster with GPUs, you can skip this step.
+
+
+Otherwise, please follow [this tutorial](kuberay-gke-gpu-cluster-setup), but substituting the following GPU node pool creation command to create a Kubernetes cluster on GKE with four Nvidia T4 GPUs:
```

Comment:
```suggestion
Otherwise, follow [this tutorial](kuberay-gke-gpu-cluster-setup), but substitute the following GPU node pool creation command to create a Kubernetes cluster on GKE with four Nvidia T4 GPUs:
```